apps/condo/domains/resident/utils/testSchema/index.js

```diff
@@ -53,13 +53,26 @@ async function updateTestResident (client, id, extraAttrs = {}) {
     const attrs = {
         dv: 1,
         sender,
-        unitName: faker.random.alphaNumeric(3),
         ...extraAttrs,
     }
     const obj = await Resident.update(client, id, attrs)
     return [obj, attrs]
 }
 
+async function softDeleteTestResident (client, id, extraAttrs = {}) {
```

Comment:
You can use update. We usually don’t create an until for that

Reply:
I think, soft-delete is a specific case, that needs to pay extra attention, is't?
BTW, we already have `softDeleteTestOrganizationEmployee`.

Reply:
it's better to add it to the core `generateServerUtils` ? to avoid code duplication?

Reply:
Later, not in this task, please.
Created a separate task SBERDOMA-965 for that.

Reply:
yes! ship it